src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int side;
+    private int base;
+    private String color = getColor();
+
+    public RightTriangle(int side, int base, String color) {
+        this.side = side;
+        this.base = base;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (double) (side * base) / 2;
```

Comment:
The expression `(double) (side * base) / 2` can potentially lead to integer overflow before the division if `side` and `base` are large enough. It's safer to cast one of the operands to `double` before the multiplication.